addons/project/tests/test_project_subtasks.py

```diff
@@ -266,3 +267,18 @@ def test_subtask_creation_with_form(self):
         self.assertEqual(task.subtask_count, 2, "Parent task should have 2 children")
         task_2 = task.copy()
         self.assertEqual(task_2.subtask_count, 2, "If the parent task is duplicated then the sub task should be copied")
+
+    def test_parent_task_editing_in_batch_mode(self):
```

Comment:
Does this test actually test what the bugfix is fixing? Because, as I understand, the error was triggering server side? Wouldn't we need a tour?